linkerd/linkerd.go

```diff
@@ -290,7 +290,7 @@ func (iClient *Client) executeInstall(ctx context.Context, arReq *meshes.ApplyRu
 	var tmpKubeConfigFileLoc = path.Join(os.TempDir(), fmt.Sprintf("kubeconfig_%d", time.Now().UnixNano()))
 	// -L <namespace> --context <context name> --kubeconfig <file path>
 	// logrus.Debugf("about to write kubeconfig to file: %s", iClient.kubeconfig)
-	if err := ioutil.WriteFile(tmpKubeConfigFileLoc, iClient.kubeconfig, 0700); err != nil {
+	if err := ioutil.WriteFile(tmpKubeConfigFileLoc, iClient.kubeconfig, 0600); err != nil {
```

Comment:
I have a consider on `gosec-G306`, Should we ignore the create file permission check? The kubeconfig file under the 0600 only can be r/d, but under 0700 can be r/d/e. I'm not sure the what's the permission of kubeconfig in this adapter completely

Reply:
Good of you to ask. I can’t think of a reason that kubeconfig would need to be executable. Unless I’m missing something, we should be ok to move to 600.